library/std/src/fs.rs

```diff
@@ -2276,6 +2289,7 @@ pub fn create_dir<P: AsRef<Path>>(path: P) -> io::Result<()> {
 ///     Ok(())
 /// }
 /// ```
+#[doc(alias = "mkdir", alias = "CreateDirectory")]
```

Comment:
Another instance where we either choose to break the alias policy, or have to decide which one of `create_dir` vs `create_dir_all` “wins out” and gets the alias.